### ISSUE TITLE
Add integration to Blend

### ISSUE THREAD

**Issue by xperiandri** [mrlacey/Rapid-XAML-Toolkit#456]
Follow the instruction here https://github.com/madskristensen/DialToolsForVS/issues/20#issuecomment-454816654

**Comment by mrlacey** [comment]
@xperiandri Which features of the toolkit are you using and which do you think would be particularly useful in Blend?

Adding Blend integration will require a lot of extra testing and possibly more work besides. Before committing to the work, I'd like to understand more about the features you're using (or want to use) and how Blend integration will benefit you (and hopefully others.)

**Comment by xperiandri** [comment]
Analyzers, paste as XAML.
It is simple. Blend is just a restricted Visual Studio instance with restricted ability to add extensions.

**Comment by mrlacey** [comment]
> It is simple

Yes, until something relied on isn't available. RXT uses some VS integration points that aren't commonly used.

Can you share more about how you've used the extension in VS and the types of projects you work on?
